### ISSUE TITLE
Documentation is not clear how to use this framework and why to use it

### ISSUE THREAD

**Issue by wojciechbulaty** [theangrydev/fluent-bdd#17]
It would be great to have a 60-90 second screen-cast:
- problem (why do I want to use it and when, show a sample end result)
- solution (how to use it, a simple example)

Also, it would be good to have a comparison in the docs, how is this framework different/better to cucumber-jvm, jbehave, fitness, yatspec, etc.

**Comment by theangrydev** [comment]
I completely agree, need to have a few different ways of explaining what the point of it is so people can decide whether it will help them or not.

My focus at the moment has been elsewhere, but it is definitely on the roadmap to get the documentation up to scratch before doing any active promotion.

So far, there is only one team of users (that I am aware of) using this, who I work with personally. At this stage, if someone new stumbles across this project, they will have to be curious enough to play around with it without much hand holding, or ask for docs like yourself.

Thanks for the prompt anyway.
